src/thermobench.cpp

```diff
@@ -203,6 +204,35 @@ void write_arr_csv(FILE *fp, double *arr, int size)
     fprintf(fp, "\n");
 }
 
+void wait_cooldown(char* exec_dir){
+
+    FILE *fp = fopen(state.sensors[0].path, "r");
+    if (fp == NULL)
+        err(1, "Error while opening sensor file: %s\n", state.sensors[0].path);
+
+    char* cmd;
+    asprintf(&cmd,"%s/fan_ctrl.sh 1", exec_dir);
```

Comment:
Do not enforce the command name. Add argument `-f, --fan-cmd` that will specify the command to run.